src/System.Threading.AccessControl/tests/System.Threading.AccessControl.Tests.csproj

```diff
@@ -16,7 +16,7 @@
     <ProjectReference Include="..\src\System.Threading.AccessControl.csproj">
       <Project>{E3ED83FD-3015-4BD8-A1B8-6294986E6CFA}</Project>
       <Name>System.Threading.AccessControl</Name>
-      <OSGroup>$(TargetOS)</OSGroup>
+      <OSGroup>$(InputOSGroup)</OSGroup>
```

Comment:
Good catch. I added the following in the root dir.props to catch this scenario for all csproj files and provide an InputOSGroup value if one does not exist:

```
<PropertyGroup>
    <InputOSGroup Condition="'$(InputOSGroup)'==''">$(OSEnvironment)</InputOSGroup>
  </PropertyGroup>
```